packages/manager/modules/notifications-sidebar/src/controller.js

```diff
@@ -31,18 +37,46 @@ export default class NotificationsCtrl {
     this.isLoading = true;
     this.numberOfActiveNotifications = 0;
 
+    // Will be bound to the click event on $document
+    this.readAllNotifications = () => {
```

Comment:
**question:** No `return` needed for this arrow function? (I'm asking this question because a promise is launched)

Reply:
No, nothing is waited in return with this function.